src/components/SessionInfoBar.tsx

```diff
@@ -34,6 +63,7 @@ export function SessionInfoBar() {
         >
           <Clipboard className="icon" />
         </a>
+        { isHealthy ? <CircleFill color="green" title="Server heartbeat status is healthy" /> : <CircleFill color="red" title="Server heartbeat status IS NOT healthy" /> }
```

Comment:
I am slightly color blind and this is messing with me. Might need to do a little gradient to make it easier to see.

Reply:
Changed icons to make it easier for myself.